rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, person
+
+  def initialize(book, person, date)
+    @date = date
+    @book = book
+    @person = person
+    book.rentals << self
+    person.rentals << self
+  end
+end
```

Comment:
- [ ]  A method for adding rentals should be present. This method should be added to add both date and person in Rentals.  for more info check this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)

Reply:
Hi @ishimwezachee ,

Thank you for detailed feedback. Could you please provide more information on what you want me to do here, I don't totally understand.